### PR TITLE
BugFix: tax_included_in_price WAS: Commit Shipping Refunds

### PR DESCRIPTION
**Pull request by glongman**
## related https://git.fullscript.io/developers/hw-admin/merge_requests/8716

### CODE REVIEW

File: app/models/spree_avatax_certified/line.rb [hunk 1]
```diff
@@ -74,6 +74,22 @@ def shipment_lines_array
       ship_lines
     end
 
+    def shipping_refund_lines_array
+      @logger.info('build shipment refund lines')
+
+      ship_refund_lines = []
+      order.shipments.each do |shipment|
+
+        if shipment.tax_category
```
Comment (lemuelbarango) [inline app/models/spree_avatax_certified/line.rb]: Does shipment have a tax_category field?

Comment (glongman) [reply app/models/spree_avatax_certified/line.rb]: It does but we don't used it. `shipping_refund_lines_array` is overridden in hw_admin

Comment (glongman) [reply app/models/spree_avatax_certified/line.rb]: I didn't want to bring the concept of master shipment into the gem. It's so ugly this thing :(

**Comment by glongman** [comment]
This PR has changed. Now only contains a bugfix. Most logic moved to the hw_admin decorator here:

https://git.fullscript.io/developers/hw-admin/merge_requests/8716

**Pull request merged**